app/models/admin.rb

```diff
@@ -0,0 +1,7 @@
+class Admin < ApplicationRecord
+  # Include default devise modules. Others available are:
```

Comment:
Avoid leaving comments in models, controllers and views.

Reply:
done